common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -319,9 +339,14 @@ def get_pending_reviews(self, user_id):
         url = self.pending_reviews_url
         student_uid = self.generate_student_uid(user_id)
         payload = {'student_uid': student_uid, 'nested': True}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        if response.content == 'No Student Found':
+        content = 'No Student Found'
+        try:
+            response = requests.get(url, headers=self.request_headers, data=payload)
+            content = response.content
+        except requests.ConnectionError:
+            log.warning('Could not connect to Adaptive Learning knowledge pending reviews url.')
```

Comment:
@macornwell Please update the message to say:

```python
"Could not get pending reviews for user identified by user_id {user_id}.".format(user_id=user_id)
```

You'll need to add line breaks as necessary.

Also, let's use a [try-except-else](https://docs.python.org/2/tutorial/errors.html#defining-clean-up-actions) here:

```python
        url = self.pending_reviews_url
        student_uid = self.generate_student_uid(user_id)
        payload = {'student_uid': student_uid, 'nested': True}
        try:
            response = requests.get(url, headers=self.request_headers, data=payload)
        except requests.ConnectionError:
            pending_reviews_user = []
            log.warning("Could not get pending reviews for user identified by user_id {user_id}.".format(user_id=user_id))
        else:
            if response.content == 'No Student Found':
                pending_reviews_user = []
            else:
                pending_reviews_user = json.loads(response.content)
        return pending_reviews_user
```